automod_legacy/swearwords.go

```diff
@@ -46,6 +49,8 @@ var BuiltinSwearWords = map[string]bool{
 	"muff":        true,
 	"nigger":      true,
 	"nigga":       true,
+	"n1gga":       true,
+	"n1gger":       true,
```

Comment:
```suggestion
	"n1gger":      true,
```
In any case, if you do wish to resubmit this to the proper branch, you will want to fix this (`gofmt` is making the CI fail currently).